src/main/resources/init_db.sql

```diff
@@ -18,3 +18,23 @@ CREATE TABLE `drivers` (
                                   PRIMARY KEY (`id`),
                                   UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
                                   UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+
+CREATE TABLE `cars` (
+                        `id` bigint NOT NULL AUTO_INCREMENT,
+                        `model` varchar(255) DEFAULT NULL,
+                        `is_deleted` tinyint NOT NULL DEFAULT '0',
+                        `manufacturer_id` bigint DEFAULT NULL,
+                        PRIMARY KEY (`id`),
+                        KEY `cars_manufacturer_fk` (`manufacturer_id`),
+                        CONSTRAINT `cars_manufacturer_fk` FOREIGN KEY (`manufacturer_id`) REFERENCES `manufacturers` (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=57 DEFAULT CHARSET=utf8mb3;
+
+CREATE TABLE `cars_drivers` (
+                                `car_id` bigint NOT NULL,
+                                `driver_id` bigint NOT NULL,
+                                `is_deleted` tinyint NOT NULL DEFAULT '0',
```

Comment:
cars_drivers shouldn't implement soft delete technique

Reply:
Ok,how i can delete all relations for some car without soft delete?

Reply:
First you do in cara is_deleted true, and then for all the data on this car from cars_driver you do exactly delete.
Just an additional method and a separate request, as in other places.